tests/test_managed_group.rs

```diff
@@ -167,19 +172,43 @@ fn managed_group_operations() {
         {
             let group_id = GroupId::from_slice(b"Test Group");
 
-            // Define credential bundles
-            let alice_credential_bundle = CredentialBundle::new(
-                "Alice".into(),
-                CredentialType::Basic,
-                ciphersuite.signature_scheme(),
-            )
-            .unwrap();
-            let bob_credential_bundle = CredentialBundle::new(
-                "Bob".into(),
-                CredentialType::Basic,
-                ciphersuite.signature_scheme(),
-            )
-            .unwrap();
+            let mut key_store = KeyStore::default();
```

Comment:
Not using `Cell` or something in the `KeyStore` of course requires you to have a mutable store. It's not used a lot but it's of course something that later needs refactoring. (Might be something to change from the beginning to safe us some refactoring.)

Reply:
Right. I'm in the process of finding a good solution for this. The goal of the next PR is to allow access via non-mutable references, such that we can actually use the `KeyPackageBundle` storage functionality.